loops_lab.js

```diff
@@ -0,0 +1,137 @@
+/*
+1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+*/
+
+let num = 15; // for #1 & #2, I wanted to combine both while and for loops, and not do them seperate. was not sure which way you wanted, but I did it.
+while ( num >= 1){
+  for ( num == 15; num > 1; num--){
+  console.log(num)
+}
+console.log(num);
+  num--;
+
+ }
+
+/*
+2. Write a `while` loop and a `for` loop that takes the variable "max", and iterates over all numbers from 0 to "max".
+For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
+*/
+console.log("----------------");
+let num2 = 0;
+while ( num2 <= 10){
+  for ( num2 == 0; num2 % 2 ===0; num2++){
+  console.log(num2 + " Even")
+  }
+if (num2 % 2 ===1){
+  console.log(num2 +" Odd");
+}
+
+
+// for (num2 == 0; num2 % 2 ===1; num2++){
+// console.log(num2 + " Odd");
+// }
+//console.log(num2);
+  num2++;
+
+}
+
+
+console.log("-------------");
+
+/*
+3. Write a `while` loop and a `for` loop that takes the variable "x" and iterates over all numbers from 0 to "x".
+For each iteration of the loop, it will multiply the number of "x" by 9 and log the result (e.g. "2 * 9 = 18")
+*/
+
+//while loop
+
+let x = 0;
+while (x <= 5){
+  console.log(x * 9);
+  x++
+}
+console.log("__________");
+// for loop;
+
+for ( let y = 3; y <= 9; y++){
+  console.log(y*9);
+}
+
+console.log("-------------");
+
+/*
+4. Write code that prints/logs all the numbers that end in a 5 from 1 to 100, **exclusive**.
+*/
+
+let number = 1;
```

Comment:
This solution works for this specific question since the question is asking to start the number from 1. But what would happen if we start from 3? Maybe we can think of a solution that uses modulo? (i.e. if( number % 5 === 0 && number % 10 !== 0)